run/GCHP/ExtData.rc.templates/ExtData.rc.carboncycle

```diff
@@ -0,0 +1,535 @@
+Ext_AllowExtrap: .true.
```

Comment:
This should be renamed to ExtData.rc.carbon.

Reply:
Thanks for catching that. This is now fixed in https://github.com/geoschem/geos-chem/pull/1605/commits/ce9b096b1d63735625f6eb1ee459d44494babd11.